pkg/ebpf/tracer.go

```diff
@@ -147,7 +148,11 @@ func NewFlowFetcher(cfg *FlowFetcherConfig) (*FlowFetcher, error) {
 	if oldKernel {
 		log.Infof("kernel older than 5.14.0 detected: not all hooks are supported")
 	}
-	objects, err := kernelSpecificLoadAndAssign(oldKernel, spec)
+	rtKernel := kernel.IsRealTimeKernel() && kernel.IsKernelOlderThan("5.14.0-292")
+	if rtKernel {
+		log.Infof("kernel is realtime and older than 5.14.0-292 not all hooks are supported")
```

Comment:
At some point it would be really awesome to get these in operator status, health dashboard and console plugin display !

Reply:
I think we need to find a way as this list now is keep growing